data/json/monstergroups.json

```diff
@@ -1219,7 +1221,24 @@
       { "monster" : "mon_zombie_brute", "freq" : 15, "cost_multiplier" : 15 },
       { "monster" : "mon_zombie_hulk", "freq" : 1, "cost_multiplier" : 50 },
       { "monster" : "mon_zombie_master", "freq" : 1, "cost_multiplier" : 30 },
-      { "monster" : "mon_zombie_hollow", "freq" : 2, "cost_multiplier" : 10 }
+      { "monster" : "mon_zombie_hollow", "freq" : 2, "cost_multiplier" : 10 },
+      { "monster" : "mon_zombie_predator", "freq" : 5, "cost_multiplier" : 10 }
+    ]
+  },{
+    "type":"monstergroup",
+    "name" : "GROUP_FERAL_HUNTER",
+    "default" : "mon_null",
+    "//" : "Masters pick from here when upgrading; no dogs, bionics, or profession-types",
+    "monsters" : [
+      { "monster" : "mon_zombie_hunter", "freq" : 20, "cost_multiplier" : 5 }
+    ]
+  },{
+    "type":"monstergroup",
+    "name" : "GROUP_FERAL_PREDATOR",
+    "default" : "mon_null",
+    "//" : "Masters pick from here when upgrading; no dogs, bionics, or profession-types",
+    "monsters" : [
+      { "monster" : "mon_zombie_predator", "freq" : 20, "cost_multiplier" : 5 }
```

Comment:
Your monster will have only a 20/1000 chance of spawning as the other 980 will be mon_null.(the remaining difference is given to the default monster which is null).  

You should set the default as mon_zombie_predator.